proposals/0001-lhls.md

```diff
@@ -0,0 +1,313 @@
+
+## What is the cost of not doing this?
+
+Low-latency streaming is becoming increasingly desired. Larger companies such as Akamai, Wowza, and Periscope offer low-latency streaming solutions. However, these solutions are not free or open. Not implementing this feature in Hls.js would make it more difficult for all video developers to access an easy and effective LHLS solution; if LHLS becomes a basic feature in the future, not having a FOSS solution will drive people towards a closed solution. Furthermore, developing LHLS in the open allows everyone to participate in its development.
+
+# Supported Environments
+[supported-environments]: #supported-environments
+
+Only browsers which support the Fetch API with streaming body responses are able to support LHLS. This includes:
+
+- Chrome >= 63
+- Edge >= 16
+- Safari >= 11.1
+- Firefox >= 57 (streaming Fetch is currently behind a flag)
+
+# Prior art
+[prior-art]: #prior-art
+
+- https://medium.com/@periscopecode/introducing-lhls-media-streaming-eb6212948bef
+- https://speakerdeck.com/stswe/cmaf-low-latency-streaming-by-will-law-from-akamai
+- https://www.wowza.com/products/capabilities/low-latency
+
+# Unresolved questions
+[unresolved-questions]: #unresolved-questions
+
+## What parts of the design do you expect to resolve through the RFC process before this gets merged?
+
+- SSAI Support
+    - SCTE-35
+- Exclusive support for CMAF (aka forbidding MPEG-TS)
+- Should the manifest only update after the prefetch frags have completed? Can prefetch frags be repeated if they are not yet completed?
+- Should playback rate and rebuffer handling be unified in this spec, or should it be up to the client implement as they see fit?
+ - At what interval should the client reload an LHLS playlist in order to minimize latency caused
+ 
+## What parts of the design do you expect to resolve through the implementation of this feature before stabilization?
+
+Aside from `DISCONTINUITY`, It is not certain which tags need a prefetch prefix to ensure backwards compatibility with players that do not support this proposal. For example, we do not know how Safari's native player will handle a rogue `EXT-X-PROGRAM-DATE-TIME` tag. When test manifest are acquired, implementers must ensure that no unexpected behavior is introduced. 
+
+## What related issues do you consider out of scope for this RFC that could be addressed in the future independently of the solution that comes out of this RFC?
+
+- Alternative connection protocols (WebRTC, Websockets, etc.)
+- Manifestless mode
```

Comment:
It might be worth linking to a resource that describes this concept.

Reply:
Yeah we can link out to some DASH docs for now, there isn't anything specified for HLS atm